### PR TITLE
Introducing computational environments to Task API

### PR DESCRIPTION
**Pull request by Wiezzel**
* CreateTaskReply: `env_id` and `prerequisites_json` added
* Constants for environment IDs
* `DockerPrerequisites` schema
* Utility functions for creating responses

### CODE REVIEW

**Comment by Wiezzel** [comment]
Not sure about the approach, let me know what you think.

File: python/golem_task_api/envs/__init__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from google.protobuf.json_format import MessageToJson
+
+from golem_task_api.envs.constants import *
```
Comment (maaktweluit) [inline python/golem_task_api/envs/__init__.py]: i personally do not like `import *` is there something in our styleguide on this?

Would propose to `import constants`

Comment (Wiezzel) [reply python/golem_task_api/envs/__init__.py]: That's intentional. I would like re-export all env-related constants. But I can enumerate them as well.

File: python/golem_task_api/envs/__init__.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+) -> CreateTaskReply:
+    prerequisites = DockerPrerequisites()
+    prerequisites.image = image
+    prerequisites.tag = tag
+    prerequisites_json = MessageToJson(prerequisites)
+
+    reply = CreateTaskReply()
+    reply.env_id = env_id
+    reply.prerequisites_json = prerequisites_json
+
```
Comment (Krigpl) [inline python/golem_task_api/envs/__init__.py]: Missing return statement?

**Pull request merged**